kubernetes/deployments/okapi-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    run: okapi-cluster
+  name: okapi-cluster
+  namespace: default
+spec:
+  type: LoadBalancer
+  loadBalancerIP: 104.196.172.62
+  ports:
+    port: 80
+    protocol: TCP
+    targetPort: 9130
+  selector:
+    run: okapi-cluster 
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    run: okapi-cluster
+  name: okapi-cluster
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: okapi-cluster
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: okapi-cluster
+    spec:
+      containers:
+      - env:
+        - name: DB_HOST
+          value: localhost
+        - name: DB_PORT
+          value: "5432"
+        - name: DB_DATABASE
+          value: folio
+        - name: DB_USER
```

Comment:
This should be `DB_USERNAME` like the others. If I remember correctly, the current custom docker file uses this var (`DB_USER`). The `okapi-initdb-and-start` script will look for `DB_USERNAME` instead of `DB_USER`.